supertokens_python/recipe/dashboard/recipe.py

```diff
@@ -119,14 +115,140 @@ def is_error_from_this_recipe_based_on_instance(self, err: Exception) -> bool:
         )
 
     def get_apis_handled(self) -> List[APIHandled]:
-        # Normally this array is used by the SDK to decide whether the recipe
-        # handles a specific API path and method and then returns the ID.
-
-        # However, for the dashboard recipe this logic is fully custom and handled inside the
-        # `return_api_id_if_can_handle_request` method of this class. Since this array is never
```

Comment:
have you tested this change with the latest dashboard? Make sure that dashboard works as expected.

Reply:
Tested and self-reviewed the code. Passed tenant-id to 2 places where it was missing.